Add docs for `{NavigateEvent,PopStateEvent}.hasUAVisualTransition` (#36705)
mdn/content: files/en-us/web/api/popstateevent/index.md

- {{domxref("PopStateEvent.state")}} {{ReadOnlyInline}}
  - : Returns a copy of the information that was provided to `pushState()` or `replaceState()`.

> - {{domxref("PopStateEvent.hasUAVisualTransition", "hashChange")}} {{ReadOnlyInline}} {{Experimental_Inline}}

Author: ```suggestion
- {{domxref("PopStateEvent.hasUAVisualTransition", "hasUAVisualTransition")}} {{ReadOnlyInline}} {{Experimental_Inline}}
```

>   - : Returns `true` if the navigation has been performed a visual transition before dispatching this event by the user agent, or `false` otherwise.

Reviewer: ```suggestion
  - : Returns `true` user agent performed a visual transition for this navigation before dispatching this event, or `false` otherwise.
```
